resteasy/src/main/java/com/cerner/beadledom/resteasy/ResteasyModule.java

```diff
@@ -7,7 +7,7 @@
 import javax.inject.Singleton;
 import javax.servlet.ServletContext;
 import org.jboss.resteasy.plugins.guice.ext.RequestScopeModule;
-import org.jboss.resteasy.plugins.interceptors.encoding.GZIPEncodingInterceptor;
+import org.jboss.resteasy.plugins.interceptors.GZIPEncodingInterceptor;
```

Comment:
I'm having a funny memory (not sure if true or not) that this class was originally here and then in one of the 3.x releases it got moved into `encoding` and now they are moving it back.